domains/anshuman.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "anshuman-dev",
+    "email": "singhanshuman216@gmail.com"
+  },
+  "record": {
+    "CNAME": "anshuman-dev.github.io/anshuman-dev"
```

```suggestion
    "CNAME": "anshuman-dev.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).